Makefile

```diff
@@ -175,3 +175,4 @@ generate-kustomize: bin/helm
 	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/serviceaccount-csi-node.yaml > ../../deploy/kubernetes/base/serviceaccount-csi-node.yaml
 	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/serviceaccount-snapshot-controller.yaml > ../../deploy/kubernetes/base/serviceaccount-snapshot-controller.yaml
 	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/snapshot-controller.yaml -f ../../deploy/kubernetes/values/snapshotter.yaml > ../../deploy/kubernetes/base/snapshot_controller.yaml
+	cd charts/aws-ebs-csi-driver && mkdir -p ../../deploy/kubernetes/cluster && cp crds/snapshot-controller.yaml ../../deploy/kubernetes/cluster/crd_snapshotter.yaml
```

Comment:
could you rename the files so they are the same in helm and in kustomize, and to disambiguate snapshot-controller.yaml

```
cd charts/aws-ebs-csi-driver && mkdir -p ../../deploy/kubernetes/cluster && cp crds/snapshot-controller-crds.yaml ../../deploy/kubernetes/cluster/snapshot-controller-crds.yaml 
```

otherwise, at a glance I got confgused what the difference between templates/snapshot-controller.yaml is and crds/snapshot-controller.yaml

Reply:
sure, 1 moment.

I could switch back `enableVolumeSnapshot` to `installCRDs` in `charts/aws-ebs-csi-driver/templates/crds.yml` for `helm` version 2 compatibility if you want

Reply:
no let's leave as is, I'm tempted to deprecate enableVolumeSnapshot in another PR :joy: . We have users who want to control:

1. whether to install snapshot controller
2. whether to install snapshot sidecar
3. whether to install snapshot crds.

IMO --skip-crds satisfies 3 . enableVolumeSnapshot is just not a good name anymore.